src/zif_abapgit_definitions.intf.abap

```diff
@@ -354,6 +358,7 @@ INTERFACE zif_abapgit_definitions
            name   TYPE string,
            text   TYPE string,
            length TYPE lvc_outlen,
+           icon   TYPE abap_bool,
```

Comment:
`icon TYPE abap_bool,`

this looks wrong, either the type is wrong, or change the field name, eg. "uses_icon" or something

Reply:
renamed to "show_icon" by @mbtools